src/V3Builtin.cpp

```diff
@@ -0,0 +1,90 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: built-in packages and classes
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2004-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include "V3Ast.h"
+#include "V3Builtin.h"
+#include "V3FileLine.h"
+#include "V3Parse.h"
+#include "V3ParseSym.h"
+
+namespace V3Builtin {
```

Comment:
Probably could change this rule now, but Verilator generally uses static class methods instead of packages.